docker-storage-setup.sh

```diff
@@ -720,6 +719,21 @@ unit: sectors
 
 ${dev}1 : start=     2048, size=  ${size}, Id=8e
 EOF
+}
+
+create_partition_parted(){
+  local dev="$1"
+  parted $dev --script mklabel dos mkpart primary 0% 100%
```

Comment:
I see in man page that one can specify unit and % is one of the supported units. so may be parted is parsing 0% and 100% properly.